helm/solr-operator/templates/deployment.yaml

```diff
@@ -37,8 +37,9 @@ spec:
       serviceAccountName: {{ include "solr-operator.serviceAccountName" . }}
       containers:
       - name: {{ .Chart.Name }}
-        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}" 
+        image: {{ .Values.image.url }} 
```

Comment:
? You have;
```diff
image:
  repository: bloomberg/solr-operator
  tag: v0.2.7
  repository: eu.gcr.io/bw-prod-platform0/solr-operator
  tag: build.17
  pullPolicy: Always
```

So this won't work?

Reply:
Are you on about my Kuber PR? I'm going to change it.